Recipely/Recipely/ProfileScreen/Cells/CellsTypes.swift

```diff
@@ -0,0 +1,12 @@
+// CellsTypes.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+
+/// Энам для типов ячеек
+enum CellsTypes {
+    case info
```

Comment:
```suggestion
    case info
```
Дока для каждого кейса